### DISCUSSION TITLE
Is there a snapshot build or any way to get a new release

### DISCUSSION THREAD

**Discussion by andrewtytula** [fredemmott/StreamDeck-Discord#45] [Q&A] [answered]
The change that lets the plugin work with multi-action was added last summer, would be nice to get a build with it, even if it is a snapshot build.

**Comment by fredemmott** [comment, marked as answer]
https://github.com/fredemmott/StreamDeck-Discord/releases/tag/v2.4rc2

I'll retag that as 2.4 now

**Reply by fredemmott** [reply]
https://github.com/fredemmott/StreamDeck-Discord/releases/tag/v2.4 - no real changes from v2.4rc2

**Comment by andrewtytula** [comment]
Just downloaded it. The multiaction works great. I was expecting it to only toggle mute, but it seems to work well with the setting the mute to on or off.
